test/api_image_search_test.go

```diff
@@ -3,6 +3,11 @@ package main
 import (
 	"github.com/alibaba/pouch/test/environment"
```

Comment:
could we format the imports here, first part is `net/url`, second part is the pouch project's imports, second is the third-party imports.